cmd/minishift/cmd/config/view_test.go

```diff
@@ -74,6 +77,53 @@ func TestConfigView(t *testing.T) {
 
 		template := determineTemplate(tt.template)
 		configView(configTest, template, tee.StdoutBuffer)
-		assert.Equal(t, tt.expectedString, tee.StdoutBuffer.String())
+		var b bytes.Buffer
+		writer := bufio.NewWriter(&b)
+		formatAndWriteLines(writer, strings.Split(tt.expectedString, "\n"))
+		expectedString := b.String()
+		assert.Equal(t, expectedString, tee.StdoutBuffer.String())
+	}
+}
+
+func TestFormatAndWriteLines(t *testing.T) {
+	type args struct {
+		lines []string
```

Comment:
I do not think the naming is very intuitive...
Can this be different?

Reply:
Renamed to alignTabbedLines, does this sound fine?